MOBILE_CLIENT/COMMON/_HIGH/TAPJACKING_VULNERABILITY/recommendation.md

```diff
@@ -0,0 +1,16 @@
+
+# Overlay Vulnerability
+
+To mitigate Tapjacking vulnerabilities in mobile applications, developers should:
+
+ * Implement robust security measures for mobile applications to mitigate overlay vulnerabilities.
```

Comment:
This has nothing to do with the finding we are reporting, we are just ensuring that a certain setting should not be set.